app/v2/controllers/actions/MessageSizeAction.scala

```diff
@@ -31,23 +31,23 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 import scala.util.Try
 
-class MessageSizeAction @Inject() (config: AppConfig)(implicit val executionContext: ExecutionContext) extends ActionRefiner[Request, Request] {
+class MessageSizeAction[R[_] <: Request[_]] @Inject() (config: AppConfig)(implicit val executionContext: ExecutionContext) extends ActionRefiner[R, R] {
```

Comment:
This change is to simply allow us to pass through the type of request we have, so we can use the authenticated request if this is later in the chain.